### PR TITLE
Added mention of complement system to thrombosis sentence

### PR DESCRIPTION
**Pull request by dziakj1**
… as @RLandon suggested in his comments.

### Suggested reviewers (optional)

@RLordan 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -152,6 +152,8 @@ Other viral infections have been associated with coagulation defects; notably SA
 A wide range of viral infections can affect the coagulation cascade.
 The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor (VWF) clotting protein, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].
 Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases [@doi:10/ggvd74; @doi:10.1186/s13054-020-03060-9; @doi:10.1007/s11239-020-02134-3].
+This excessive clotting in lung capillaries has been suggested to related to a dysregulated activation of the complement system, part of the innate immune system [@doi:10.1172/jci.insight.140711;@doi:10.1172/JCI140183].
```
Comment (rando2) [inline content/02.introduction.md:155]:
```suggestion
This excessive clotting in lung capillaries has been suggested to be related to dysregulated activation of the complement system, part of the innate immune system [@doi:10.1172/jci.insight.140711; @doi:10.1172/JCI140183].
```

Comment (rando2) [reply content/02.introduction.md:155]: @yemarshall the first reference says that it's "at the interface" of innate and adaptive, the second says it's part of the innate immune system -- thought I'd ask for your thoughts!

**Comment by rando2** [comment]
One more thing to add -- I think you might have meant to tag @RLordan here, so just in case Ronan is interested, I'm tagging him!

**Comment by RLordan** [comment]
> One more thing to add -- I think you might have meant to tag @RLordan here, so just in case Ronan is interested, I'm tagging him!

Thanks @rando2 I always referred to the complement as part of the innate immune system but its functions are widespread so hearing that it is at the interface doesn't surprise me. Definitely have someone more knowledgeable of immunology to confirm :)

**Pull request merged**